src/index.ts

```diff
@@ -58,20 +59,34 @@ async function main() {
         workflow_id,
         branch,
       });
+      console.log(`Found ${data.total_count} runs total.`);
 
-      const branchWorkflows = data.workflow_runs.filter(run => run.head_branch === branch);
-      console.log(`Found ${branchWorkflows.length} runs for workflow ${workflow_id} on branch ${branch}`);
-      console.log(branchWorkflows.map(run => `- ${run.html_url}`).join('\n'));
```

Comment:
I think the PR would be easier to review if this change was reverted and the filter below was not done in one big combination, but done separately as before. Then this whole part of the diff would disappear and the filter below would be simpler? That said, it doesn't look incorrect